core/src/raw/http_util/header.rs

```diff
@@ -199,6 +199,21 @@ pub fn parse_content_disposition(headers: &HeaderMap) -> Result<Option<&str>> {
     }
 }
 
+/// Parse header value to string.
+pub fn parse_header_to_str(headers: &HeaderMap, name: String) -> Result<Option<&str>> {
```

Comment:
How about accepting a `HeaderName` or `impl Into<HeaderName>`?

Reply:
Perhaps we should use the same function signature as `HeaderMap.get`.
```rust
pub fn parse_header_to_str<K: AsHeaderName>(headers: &HeaderMap, name: K) -> Result<Option<&str>> {
```

Reply:
LGTM

Reply:
fixed